test/resources/knativeserving.go

```diff
@@ -118,13 +118,15 @@ func getDeploymentStatus(d *v1.Deployment) corev1.ConditionStatus {
 
 func getTestKSOperatorCRSpec() v1alpha1.KnativeServingSpec {
 	return v1alpha1.KnativeServingSpec{
-		Config: map[string]map[string]string{
-			DefaultsConfigKey: {
-				"revision-timeout-seconds": "200",
-			},
-			LoggingConfigKey: {
-				"loglevel.controller": "debug",
-				"loglevel.autoscaler": "debug",
+		CommonSpec: v1alpha1.CommonSpec{
```

Comment:
Are we willing to accept this change?

Reply:
I'm ok with it. It's only an issue for literals, so just makes tests a little noisier. But I'm willing to trade that for the elimination of the duplicate fields.

Reply:
@markusthoemmes Do we change the structure in CR yamls?
Based on my understanding, it is no. That is fine.
CommonSpec is included directly in KnativeServingSpec and KnativeEventingSpec. That looks find to me as well.

Reply:
Yeah, as Jim mentioned this is only about literals in Golang.